standard/expressions.md

```diff
@@ -5814,7 +5814,22 @@ When a property or indexer declared in a *struct_type* is the target of an assig
 
 If the left operand of a compound assignment is of the form `E.P` or `E[Ei]` where `E` has the compile-time type `dynamic`, then the assignment is dynamically bound ([§11.3.3](expressions.md#1133-dynamic-binding)). In this case, the compile-time type of the assignment expression is `dynamic`, and the resolution described below will take place at run-time based on the run-time type of `E`. If the left operand is of the form `E[Ei]` where at least one element of `Ei` has the compile-time type `dynamic`, and the compile-time type of `E` is not an array, the resulting indexer access is dynamically bound, but with limited compile-time checking ([§11.6.5](expressions.md#1165-compile-time-checking-of-dynamic-member-invocation)).
 
-An operation of the form `x «op»= y` is processed by applying binary operator overload resolution ([§11.4.5](expressions.md#1145-binary-operator-overload-resolution)) as if the operation was written `x «op» y`. Then,
+For operator `??=`, given `a ??= b`, where `A` is the type of `a`, `B` is the type of `b`, and `A0` is the underlying type of `A`, if `A` is a nullable value type:
+
+1. If `A` does not exist or is a non-nullable value type, a compile-time error occurs.
+2. If `B` is not implicitly convertible to `A` or `A0` (if `A0` exists), a compile-time error occurs.
```

Comment:
Hmm, implicit integer literal value-based conversions... so you're think of something along the lines of:

> If `B` is not implicitly convertible to `A` or `A0` (if `A0` exists) _or_ if `b` is a constant expression of type `int` and its value is not implicitly  convertible to `A` or `A0` (if `A0` exists), then a compile-time error occurs.

I think that wording captures both int literal to smaller integral types and int literal zero to enum types.

Ouch, I can't but wonder how many times that might be missed in the Standard...

This might require an issue of its own rather than fixing it when it is spotted – unless someone already knows it is allowed for wherever needed?